internal/pkg/cli/task_run.go

```diff
@@ -297,14 +310,88 @@ func (o *runTaskOpts) Execute() error {
 		}
 	}
 
-	_, err := o.runTask()
+	tasks, err := o.runTask()
 	if err != nil {
 		return err
 	}
 
+	if o.follow {
+		if err := o.displayLogStream(tasks); err != nil {
+			return err
+		}
+	}
 	return nil
 }
 
+func (o *runTaskOpts) displayLogStream(tasks []*task.Task) error {
+	startTime := task.EarliestStartTime(tasks)
+
+	logGroupName := fmt.Sprintf(fmtTaskLogGroupName, o.groupName)
+	logEventsOutput := &cloudwatchlogs.LogEventsOutput{
+		LastEventTime: make(map[string]int64),
+	}
+	for {
+		for i := 1; i < numCWLogsCallsPerRound; i++ {
```

Comment:
hmm should this be i:=0 so that per round is 10 otherwise it is actually 9...

Reply:
lol..........funny why i typed 1